inmoose/utils/format.py

```diff
@@ -0,0 +1,45 @@
+# -----------------------------------------------------------------------------
+# Copyright (C) 2022-2023 M. Colange
```

Comment:
credit the actual author(s) of the code
that would be "Copyright (C) 2024 L. Meunier"